airflow/scripts/git_scripts/aggregate_repo_activity.py

```diff
@@ -0,0 +1,85 @@
+
+import requests
+from collections import defaultdict
```

Comment:
Order the imports alphabetically, starting with local imports, followed by third-party packages, and ending with custom functions.
Example:
from collections import defaultdict
from datetime import datetime, timedelta

import requests

import fetch_repo_data